src/main/resources/db/migration/V14__video_link_booking_unique_constraints.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE video_link_booking ADD UNIQUE (pre_appointment);
```

Comment:
These constraints express properties of the data and add indexes on pre,main,post appointment. 
Do they help? Are they superfluous?